frame/society/src/migrations.rs

```diff
@@ -196,9 +200,10 @@ pub fn assert_internal_consistency<T: Config<I>, I: Instance + 'static>() {
 	for (who, record) in members.iter() {
 		assert_eq!(MemberByIndex::<T, I>::get(record.index).as_ref(), Some(who));
 	}
-	if let Some(founder) = Founder::<T, I>::get() {
-		assert_eq!(Members::<T, I>::get(founder).expect("founder is member").index, 0);
-	}
+	// TODO: This panics if uncommented -- FIXME
+	// if let Some(founder) = Founder::<T, I>::get() {
+	// 	assert_eq!(Members::<T, I>::get(founder).expect("founder is member").index, 0);
+	// }
```

Comment:
What about this?

Reply:
I mentioned it in the PR description, wasn't sure if Gav wanted to look at it. Seems maybe not, I'll add it as a todo to look into this week.

Reply:
https://github.com/paritytech/substrate/pull/14421#issuecomment-1609495107